src/template.json

```diff
@@ -34,41 +37,21 @@
   ],
   "provisioners": [
     {
-      "environment_vars": [
-        "DEVOPS_LIFE_IMPROVER=PACKER"
-      ],
       "inline": [
-        "Write-Host \"HELLO NEW USER; WELCOME TO $Env:DEVOPS_LIFE_IMPROVER\"",
-        "Write-Host \"You need to use backtick escapes when using\"",
-        "Write-Host \"characters such as DOLLAR`$ directly in a command\"",
-        "Write-Host \"or in your own scripts.\""
-      ],
-      "type": "powershell"
-    },
-    {
-      "type": "windows-restart"
-    },
-    {
-      "environment_vars": [
-        "VAR1=A$Dollar",
-        "VAR2=A`Backtick",
-        "VAR3=A'SingleQuote",
-        "VAR4=A\"DoubleQuote"
+        "C:/ProgramData/Amazon/EC2-Windows/Launch/Scripts/InitializeInstance.ps1 -Schedule",
+        "C:/ProgramData/Amazon/EC2-Windows/Launch/Scripts/SysprepInstance.ps1 -NoShutdown"
```

Comment:
I don't see anything wrong with this, but it is unclear why this change is happening, can you spell it out?

Reply:
The things remove are not needed.  i believe @thomasyu888 was still experimenting and testing so he left that cruft around. As for  InitializeInstance.ps1 and SysprepInstance.ps1.  Those were in the packer examples and i believe the reason why you need them is that it will allow all new Instances launched off this AMI to decrypt the password from your AWS Console. here's some more info about it: https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2launch.html#Schedule-EC2Launch-to-run-on-every-boot

Reply:
👍